### PR TITLE
Update notebooks

### PR DESCRIPTION
**Pull request by ariematsliah-princeton**
### Description
- Cleaning up and shortening
- Updating content
- Making references more robust
- Add colab links from GUI

### Types of changes

- [ ] Bugfix
- [ ] New feature
- [ ] Refactor / Code style update (no logical changes)
- [ ] Build / CI changes
- [x] Documentation Update
- [ ] Other (explain)

### Does this address any currently open issues?
https://github.com/murthylab/sleap/issues/514

### CODE REVIEW

File: docs/guides/remote.rst [hunk 1]
```diff
@@ -33,13 +33,13 @@ When you run training, specify the full path to the :code:`json` file.
 
 **Command-line training**:
 
-Once you have your training package (or labels project file) and training profile, you can run training like so:
+Once you have your training job package (or labels package and training profile), you can run training like so:
 
 ::
 
-  sleap-train path/to/your/training_profile.json another/path/to/training_package.h5
+  sleap-train path/to/your/training_profile.json another/path/to/labels_package.h5
```
Comment (talmo) [inline docs/guides/remote.rst:40]: `labels.pkg.slp`, not `.h5` (this is old)

File: docs/notebooks/Training_with_custom_hyperparameters.ipynb [hunk 2]
```diff
@@ -300,11 +300,11 @@
         "id": "KQhv_gsdJzaq"
       },
       "source": [
-        "I'll assume you already know how to train using a training package on Colab. If you haven't done this already, you should take a look at our [other notebook](https://colab.research.google.com/github/murthylab/sleap/blob/main/docs/notebooks/Training_and_inference_using_Google_Drive.ipynb) which walks you through that process in more detail.\n",
+        "We'll assume you already know how to train using a training job package on Colab. If you haven't done this already, you should take a look at our [other notebook](https://colab.research.google.com/github/murthylab/sleap/blob/main/docs/notebooks/Training_and_inference_using_Google_Drive.ipynb) which walks you through that process in more detail.\n",
```
Comment (talmo) [inline docs/notebooks/Training_with_custom_hyperparameters.ipynb:303]: We don't want to just link to the rendered page on sleap.ai?

File: sleap/gui/app.py [hunk 3]
```diff
@@ -751,6 +751,20 @@ def new_instance_menu_action():
             "proofreading or reproducibility."
         )
 
+        predictionMenu.addSeparator()
+        add_menu_item(
+            predictionMenu,
+            "training on colab with sample dataset",
+            "Train in Google Colab with sample data..",
```
Comment (talmo) [inline sleap/gui/app.py:758]: Probably don't need a link to sample data example -- but could maybe point to https://sleap.ai/notebooks/index.html?

Comment (talmo) [reply sleap/gui/app.py:758]: Better yet: link to the training with default profiles (`Training_and_inference_using_Google_Drive.ipynb`) and training with custom profile (`Training_with_custom_hyperparameters.ipynb`)

File: docs/notebooks/Training_with_custom_hyperparameters.ipynb [hunk 4]
```diff
@@ -340,7 +340,7 @@
         "id": "zDbIRjlw_fOF"
       },
       "source": [
-        "The line below assumes your custom training profile is called `multi_instance.json` and that you've copied it into the `sleap-custom-training` directory in your Google Drive, along side your training package called `dataset.pkg.slp`. Edit the filenames in the command to match your files."
+        "The line below assumes your custom training profile is called `multi_instance.json` and that you've copied it into the `sleap-custom-training` directory in your Google Drive, along side your .slp file from training job package called `dataset.pkg.slp`. Edit the filenames in the command to match your files."
```
Comment (talmo) [inline docs/notebooks/Training_with_custom_hyperparameters.ipynb:343]: Why aren't we using the zip file-based workflow here?

It's being used in the other notebook, but it's actually _more_ relevant here.

The notebook with defaults params doesn't even use the other files in the zip file, but here we actually use the json files that specify these training configs.

File: docs/notebooks/Training_and_inference_using_Google_Drive.ipynb [hunk 5]
```diff
@@ -1,543 +1,300 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ApaDWxW4dLMS"
+   },
+   "source": [
+    "## Upload training job package to Google Drive\n",
+    "To be consistent with the examples in this notebook, name the SLEAP project `colab` and create a directory called `sleap` in the root of your Google Drive. Then upload the exported training job package `colab.slp.training_job.zip` into `sleap` directory.\n",
```
Comment (talmo) [inline docs/notebooks/Training_and_inference_using_Google_Drive.ipynb:76]: See comment in the next notebook, but this is a little weird here... when you export the package in the zip file, it includes the `.pkg.slp` file, as well as the training jobs, which will just be ignored here?

Might it make more sense to be explicit about this and/or just tell people to use the .pkg.slp file (also exportable without needing to go into the training editor)?

File: sleap/gui/app.py (hunk 3 above)
Comment (ariematsliah-princeton) [reply sleap/gui/app.py]: Good point - done

File: docs/notebooks/Training_with_custom_hyperparameters.ipynb (hunk 2 above)
Comment (ariematsliah-princeton) [reply docs/notebooks/Training_with_custom_hyperparameters.ipynb]: Done

File: docs/guides/remote.rst (hunk 1 above)
Comment (ariematsliah-princeton) [reply docs/guides/remote.rst]: Done

File: docs/notebooks/Training_with_custom_hyperparameters.ipynb (hunk 4 above)
Comment (ariematsliah-princeton) [reply docs/notebooks/Training_with_custom_hyperparameters.ipynb:343]: I didn't work on this notebook in this PR - let me update it as well now

**Pull request merged**